lib/libp2p_port.ex

```diff
@@ -530,6 +539,49 @@ defmodule LambdaEthereumConsensus.Libp2pPort do
     {:noreply, state}
   end
 
+  @impl GenServer
+  def handle_call(:get_keystores, _from, %{validators: []} = state),
+    do: {:reply, [], state}
+
+  @impl GenServer
+  def handle_call(:get_keystores, _from, %{validators: validators} = state),
+    do: {:reply, Enum.map(validators, fn {_pk, validator} -> validator.keystore end), state}
+
+  @impl GenServer
+  def handle_call({:delete_validator, pk}, _from, %{validators: validators} = state) do
+    case Map.fetch(validators, pk) do
+      {:ok, validator} ->
+        Logger.warning("[Libp2pPort] Deleting validator with index #{inspect(validator.index)}.")
+
+        {:reply, :ok, %{state | validators: Map.delete(validators, pk)}}
+
+      :error ->
+        {:error, "Pubkey #{inspect(pk)} not found."}
+    end
+  end
+
+  @impl GenServer
+  def handle_call({:add_validator, keystore}, _from, %{validators: validators} = state) do
+    # TODO (#1263): handle 0 validators
+    first_validator = validators |> Map.values() |> List.first()
+    validator = Validator.new({first_validator.slot, first_validator.root, keystore})
```

Comment:
Nothing to be done in this PR, but I think this shows how weird it is to have these values tied to individual validators. We'll get those values from the store when the other PR is merged.

Reply:
+1